tests/test_docx_adapter.py

```
@@ -0,0 +1,205 @@
+        mock_zip.read.return_value = document_xml.encode('utf-8')
+
+        with patch("zipfile.ZipFile", return_value=mock_zip), \
+             patch("pathlib.Path.exists", return_value=True):
+            segments = self.adapter.extract_segments("test.docx")
+
+            assert len(segments) == 3
+            # First segment
+            assert segments[0].id == "word/document.xml:0:0"
+            assert segments[0].text == "Hello"
+            assert segments[0].segment_type == SegmentType.PARAGRAPH
+            assert segments[0].has_japanese == False
```

Ruff (0.13.1)

Avoid equality comparisons to `False`; use `not segments[0].has_japanese:` for false checks

Replace with `not segments[0].has_japanese`

(E712)